source/extensions/common/wasm/v8/v8.cc

```diff
@@ -247,9 +249,28 @@ bool V8::load(const std::string& code, bool /* allow_precompiled */) {
   ::memcpy(source_.get(), code.data(), code.size());
 
   module_ = wasm::Module::make(store_.get(), source_);
+  if (module_) {
+    shared_module_ = module_->share();
+    RELEASE_ASSERT(shared_module_ != nullptr, "");
```

Comment:
nit: in general we don't release ASSERT things like this since it shouldn't fail and will crash in obvious ways. Is there any reason to do this here and elsewhere?

Reply:
Done.